parm/config/config.resources

```diff
@@ -746,102 +772,110 @@ elif [[ ${step} = "eobs" || ${step} = "eomg" ]]; then
-      if [[ ${machine} = "S4" ]]; then
+      elif [[ ${machine} = "S4" ]]; then
          export npe_eupd=160
          export nth_eupd=2
       fi
     elif [[ ${CASE} = "C192" || ${CASE} = "C96" || ${CASE} = "C48" ]]; then
       export npe_eupd=42
       export nth_eupd=2
-      if [[ "${machine}" = "HERA" ]]; then
+      if [[ "${machine}" = "HERA" || "${machine}" = "JET" ]]; then
         export nth_eupd=4
       fi
     fi
-    export npe_node_eupd=$(echo "${npe_node_max} / ${nth_eupd}" | bc)
+    npe_node_eupd=$(echo "${npe_node_max} / ${nth_eupd}" | bc)
+    export npe_node_eupd
     export is_exclusive=True
 
-elif [ ${step} = "ecen" ]; then
+elif [[ ${step} = "ecen" ]]; then
 
     export wtime_ecen="00:10:00"
     export npe_ecen=80
     export nth_ecen=4
     if [[ "${machine}" = "HERA" ]]; then export nth_ecen=6; fi
     if [[ ${CASE} = "C384" || ${CASE} = "C192" || ${CASE} = "C96" || ${CASE} = "C48" ]]; then export nth_ecen=2; fi
-    export npe_node_ecen=$(echo "${npe_node_max} / ${nth_ecen}" | bc)
+    npe_node_ecen=$(echo "${npe_node_max} / ${nth_ecen}" | bc)
+    export npe_node_ecen
     export nth_cycle=${nth_ecen}
-    export npe_node_cycle=$(echo "${npe_node_max} / ${nth_cycle}" | bc)
+    npe_node_cycle=$(echo "${npe_node_max} / ${nth_cycle}" | bc)
+    export npe_node_cycle
     export is_exclusive=True
 
-elif [ ${step} = "esfc" ]; then
+elif [[ ${step} = "esfc" ]]; then
 
     export wtime_esfc="00:06:00"
     export npe_esfc=80
     export nth_esfc=1
-    export npe_node_esfc=$(echo "${npe_node_max} / ${nth_esfc}" | bc)
+    npe_node_esfc=$(echo "${npe_node_max} / ${nth_esfc}" | bc)
+    export npe_node_esfc
     export nth_cycle=${nth_esfc}
-    export npe_node_cycle=$(echo "${npe_node_max} / ${nth_cycle}" | bc)
+    npe_node_cycle=$(echo "${npe_node_max} / ${nth_cycle}" | bc)
+    export npe_node_cycle
     export memory_esfc="80GB"
 
-elif [ ${step} = "epos" ]; then
+elif [[ ${step} = "epos" ]]; then
 
     export wtime_epos="00:15:00"
     export npe_epos=80
     export nth_epos=4
     if [[ "${machine}" == "HERA" ]]; then
       export nth_epos=6
     fi
-    export npe_node_epos=$(echo "${npe_node_max} / ${nth_epos}" | bc)
+    npe_node_epos=$(echo "${npe_node_max} / ${nth_epos}" | bc)
+    export npe_node_epos
     export is_exclusive=True
 
-elif [ ${step} = "postsnd" ]; then
+elif [[ ${step} = "postsnd" ]]; then
 
     export wtime_postsnd="02:00:00"
     export npe_postsnd=40
     export nth_postsnd=8
     export npe_node_postsnd=10
     export npe_postsndcfp=9
     export npe_node_postsndcfp=1
-    if [[ "$(echo "${npe_node_postsnd} * ${nth_postsnd}" | bc)" -gt "${npe_node_max}" ]]; then
-        export npe_node_postsnd=$(echo "${npe_node_max} / ${nth_postsnd}" | bc)
+    postsnd_req_cores=$(echo "${npe_node_postsnd} * ${nth_postsnd}" | bc)
+    if [[ ${postsnd_req_cores} -gt "${npe_node_max}" ]]; then
+        npe_node_postsnd=$(echo "${npe_node_max} / ${nth_postsnd}" | bc)
+        export npe_node_postsnd
```

Comment:
Can you explain this change a bit? Is this stemming from a Jet need?

Reply:
This was to address two `SC2155` shellcheck warnings -- one for the conditional and another for the export.  For the export, since `echo "${npe_node_max} / ${nth_postsnd}" | bc` is being evaluated in an assignment, if this is exported, shellcheck claimed that any errors resulting in the `bc` calculation may not cause a non-zero return status.